lib/rules/no-tryinvoke.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const types = require('../utils/types');
+const { getImportIdentifier } = require('../utils/import');
+
+const ERROR_MESSAGE = 'Use optional chaining operator `?.()` instead of `tryInvoke`';
+
+module.exports = {
+  ERROR_MESSAGE,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: "disallow usage of the Ember's `tryInvoke` util",
+      category: 'Ember Utils',
```

Comment:
Let's put this in the existing `Ember Object` since it's related to other rules in that category regarding how to access data from objects.